types/event.model.ts

```diff
@@ -0,0 +1,40 @@
+import { Crew, Team } from './crew.model';
+import { DisplayStatus } from './display-status.model';
+import { Round } from './round.model';
+
+export interface Event {
+	id: string;
+	regatta_id: string;
+	cost_id: string;
+	parent: null; // Wat is de 'parent'?
+	event: number;
+	number: number; // Is dit hetzelfde als 'event'?
+	day: Date; // Wordt een event op alleen een dag gedaan? Is dat anders dan de regatta?
```

Comment:
Officieel mag een event maar op 1 dag in 1 block gedaan worden.

Dus als de regatta meer daags is dan is een event 1 daags. Dit geld niet voor verschillende afstanden in een meerkamp.